src/skin/imgloader.cpp

```diff
@@ -11,6 +11,7 @@ ImgLoader::ImgLoader() {
 QImage* ImgLoader::getImage(const QString& fileName, double scaleFactor) const {
     QImage* pImage = new QImage();
     QFileInfo info(fileName);
+#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
     if (scaleFactor > 2.0) {
```

Comment:
`QImageReader` doesn't handle it properly and I afraid it can lead to UI artifacts.